### PR TITLE
WSTEAMA-423: Adds Page Title to atiAnalytics object to standardise schema for remaining page types

### PR DESCRIPTION
**Pull request by pvaliani**
Resolves [#WSTEAMA-423](https://jira.dev.bbc.co.uk/browse/WSTEAMA-423)

This PR reintroduced the `pageTitle` (formerly `title`) to being within the `atiAnalytics` `metadata` object. The duplication of the usage of title is necessary here to standardise the schema for other page types that will follow the `genericPage` strategy pattern.

The corresponding BFF change for this PR is:

https://github.com/bbc/fabl-modules/pull/4743 

Where the FABL `simorgh-bff` module response has also received a similar change

### CODE REVIEW

File: src/app/pages/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ const HomePage = ({ pageData }: HomePageProps) => {
         entities={[itemList]}
       />
       <main css={styles.main}>
-        <ATIAnalytics atiData={{ ...metadata, title }} />
+        <ATIAnalytics atiData={metadata} />
```
Comment (karinathomasbbc) [inline src/app/pages/HomePage/HomePage.tsx]: This is a weee bit confusing for me - I would rather that we explicitly pass `metadata.atiAnalytics` here... or rename the `metadata` prop above to `atiAnalytics`?

File: src/app/components/ATIAnalytics/params/genericPage/buildParams.ts [hunk 2]
```diff
@@ -11,27 +11,35 @@ export const buildPageATIParams = ({
...
     producerId: atiAnalyticsProducerId,
     contentType,
     pageIdentifier,
-    pageTitle,
     platform,
     statsDestination,
     service,
     timePublished,
     timeUpdated,
+    pageTitle,
```
Comment (eagerterrier) [inline src/app/components/ATIAnalytics/params/genericPage/buildParams.ts]: Why this now not alphabetical?

File: src/app/components/ATIAnalytics/params/genericPage/buildParams.ts [hunk 3]
```diff
@@ -11,27 +11,35 @@ export const buildPageATIParams = ({
...
+    pageIdentifier,
+    timePublished,
+    timeUpdated,
+    pageTitle,
   } = atiData;
+  // const {
+  //   atiAnalytics: {
+  //     contentId,
+  //     contentType,
+  //     pageIdentifier,
```
Comment (eagerterrier) [inline src/app/components/ATIAnalytics/params/genericPage/buildParams.ts]: Needed?

**Review by eagerterrier — COMMENTED** [review]
Couple of minor potential tidy ups, otherwise lgtm

File: src/app/components/ATIAnalytics/params/genericPage/buildParams.ts (hunk 3 above)
Comment (alex-magana) [reply src/app/components/ATIAnalytics/params/genericPage/buildParams.ts]: it's not. Thanks for catching this.
This had been commented out as part of running tests following a rewrite
to reference `atiAnalytics` properties.

File: src/app/components/ATIAnalytics/params/genericPage/buildParams.ts (hunk 2 above)
Comment (alex-magana) [reply src/app/components/ATIAnalytics/params/genericPage/buildParams.ts]: An oversight while renaming the `title` alias.
This has been changed.

**Review by Isabella-Mitchell — APPROVED** [review]
It's a little out of my wheelhouse - but it looks good to me based on the code and the changes you've made following other people's comments.

**Comment by paruchurisilpa** [comment]
Testing:
* Checked that https://localhost:7080/kyrgyz and ati is working as expected..

**Review by karinathomasbbc — APPROVED** [review]
🚢

**Pull request merged**